EdvDocument.js

```diff
@@ -77,20 +78,21 @@ export class EdvDocument {
   }
 
   /**
+   * @typedef Readable
```

Comment:
I like these type defs, but I believe this should be `ReadableStream` and I don't believe you need the typedef unless `ReadableStream` is not imported in this file.